contracts/marketplace/src/execute.rs

```diff
@@ -859,11 +869,15 @@ fn payout(
     Ok(())
 }
 
-fn price_validate(price: &Coin) -> Result<(), ContractError> {
+fn price_validate(store: &dyn Storage, price: &Coin) -> Result<(), ContractError> {
     if price.amount.is_zero() || price.denom != NATIVE_DENOM {
         return Err(ContractError::InvalidPrice {});
     }
 
+    if price.amount < SUDO_PARAMS.load(store)?.min_price {
+        return Err(ContractError::PriceTooSmall(price.amount));
+    }
```

Comment:
we might be loading twice the sudo params if the method caller already did  not really a problem tho.

Reply:
Yeah can load twice sometimes. I don't think its a particularly expensive task to load a single `Item`.